### PR TITLE
Fix image resize quality

### PR DESCRIPTION
**Pull request by qmeister**
Fixes issue due to reduce in quality after resize.

Original image: 
image-original

Image quality loss after resize: 
image-quality-reduced

### CODE REVIEW

**Review by martijngastkemper — COMMENTED** [review]
Dit zal het allicht oplossen, maar krijg je nu geen enorme bestanden?

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+const dotenv = require("dotenv");
+const express = require("express");
+const fs = require("fs");
```
Comment (martijngastkemper) [inline server/index.js:3]: You introduce the local test server to version 1 so these dependencies must be added as well. In versions that already have the local test server only "dotenv" must be added as dev dependency.

Comment (martijngastkemper) [reply server/index.js:3]: And the information must be added to README.md

Comment (martijngastkemper) [reply server/index.js:3]: You can also decide to first create a PR to add the test server to version 1. And continue from there to reduce the amount of conflicts.

Comment (qmeister) [reply server/index.js:3]: I've cherry-picked the addition of the local image server to add the test server to v1.0.

**Comment by qmeister** [comment]
> Dit zal het allicht oplossen, maar krijg je nu geen enorme bestanden?
_Translation: This will probably solve it, but will this make huge files now?_

File size will not be larger than original size, but it does of course effects file size of the resized image, as for now image quality will decrease with a default setting of `quality: 80`.
We've resolved this by adding this quality setting as an environment variable to make this configurable. If it has not been set it will default to 80, which is the current setting.

File: server/index.js (hunk 1 above)
Comment (martijngastkemper) [reply server/index.js:3]: Only the `dotenv` dev requirement is still missing.

**Review by martijngastkemper — COMMENTED** [review]
The local server doesn't work. It's unable to parse the dimensions.

Screenshot 2023-05-11 at 13 54 45

**Comment by qmeister** [comment]
> The local server doesn't work. It's unable to parse the dimensions.

Because of SCALED_FOLDER variable, which was still used in v1.0. Has been fixed.

**Review by martijngastkemper — APPROVED** [review]
LGTM!

**Pull request merged**